views/works/item.haml

```diff
@@ -30,22 +30,22 @@
     - puts metrics
     .panel-footer
       - unless views < 1
-        %span
+        %span.metrics-views
```

Comment:
This was part of the old implementation. Back then the icons and metrics would disappear completely if there were no metrics. However, the new mockups https://datacite.atlassian.net/wiki/spaces/TEC/pages/531529771/Display+stats+in+DataCite+Search  the icons stay present and the values change to "--". And @daslerr to clarify the "--" value should appear only when there is no data, but if a resource has 20 views but 0 downloads, it should say 0 not "--", right?

Reply:
We can do a check for something more specific, or alternativly we add a class when there is values, I kinda prefer that tbh as you could arguably then apply a style specifically to if there is no values.
So maybe .metrics-views-empty .metrics-views-notempty or something.
Food for thought, either way the actual testing shouldn't make any difference.